d3a_api_client/utils.py

```diff
@@ -47,6 +48,23 @@ def _get_request(self, endpoint_suffix, data):
         return data["transaction_id"], get_request(endpoint, data, self.jwt_token)
 
 
+def post_graphql_request(domain_name, query, headers=None, url=None, authenticate=True):
```

Comment:
```suggestion
def execute_graphql_request(domain_name, query, headers=None, url=None, authenticate=True):
```
As using `post` gives an impression that it would only be a `mutation` and no `query`